libqtile/layout/xmonad.py

```diff
@@ -227,7 +230,7 @@ def cmd_normalize(self, redraw=True):
 
     def cmd_reset(self, redraw=True):
         "Reset Layout."
-        self.ratio = self._med_ratio
+        self.ratio = self._reset_ratio = 0.5
```

Comment:
ops this should be `self.ratio = self._reset_ratio` instead of `self.ratio = self._reset_ratio = 0.5`.
I don't remember adding ` = 0.5`